src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -240,3 +240,11 @@ export function createApiAwsSamDebugConfig(
         ...withRuntime,
     }
 }
+
+function makeWorkspaceRelativePath(folder: vscode.WorkspaceFolder | undefined, target: string): string {
+    if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length <= 1) {
+        return folder ? `\${workspaceFolder}/${getNormalizedRelativePath(folder.uri.fsPath, target)}` : target
```

Comment:
If `vscode.workspace.workspaceFolders` is falsey, it should be safe to use `${workspaceFolder}` anyway, no? Perhaps that's more future-proof, in case the user changes their project layout later?

I would expect VSCode to allow users to always use `${workspaceFolder}` without needing to think about whether they actually have a workspace, it should "just work".

Reply:
Confirmed that we fail if the user isn't in a workspace: https://github.com/aws/aws-toolkit-vscode/blob/0507ee52790daeab1d8578e7e346f280ec98ed6c/src/shared/debug/launchConfiguration.ts#L107 

VS Code throws an error, we pop the toast, and an input box appears to select a folder to make a workspace from that as far as I can tell does nothing? Unsure if said box is from VS Code or us; regardless, I think this is moot for now.

![image](https://user-images.githubusercontent.com/29374703/108872957-91285c00-75af-11eb-97a6-f85beb530270.png)